api/genomics/docs/wgs_cohort_extraction_setup.md

```diff
@@ -0,0 +1,47 @@
+## Environment Setup for WGS cohort extraction
+
+1. Create wgs-cohort-extraction service account.   
+    - This should match the value in WorkbenchConfig.wgsCohortExtraction.serviceAccount
+    - This will have to go through sysadmins for non test environments
+
+2. Register service account as a Terra user 
+     1. Grab sa-creds.json for the service account. I used gcloud admin UI.
+     2. ```
+        curl -H "$(oauth2l header --json sa-creds.json userinfo.email userinfo.profile cloud-billing)" \
```

Comment:
You can replace step 1 and this header with `-H "Authorization: Bearer $(gcloud auth --impersonate-service-account EXTRACTION_SA print-access-token)"`

this also eliminates the need for the step 3 which is missing here - which would have been to delete the key you created